src/site/content/en/metrics/cls/index.md

```diff
@@ -384,6 +384,8 @@ few guiding principles:
 For a deep dive on how to improve CLS, see [Optimize
 CLS](https://web.dev/optimize-cls/).
 
+Also, consider this guide specific to [minimizing CLS when working with Google Publisher Tags (GPT)](https://developers.google.com/doubleclick-gpt/guides/minimize-layout-shift).
+
```

Comment:
```suggestion
For a deep dive on how to improve CLS, see [Optimize
CLS](https://web.dev/optimize-cls/).

### Google Publisher Tags

See the official Google Publisher Tag guide on [Minimizing layout shift](https://developers.google.com/doubleclick-gpt/guides/minimize-layout-shift).

```
I like to chunk things up into sections so that, if someone is scanning with the table-of-contents, they can quickly home in on the GPT section (if that is what's most relevant to them)